### DISCUSSION TITLE
Commands

### DISCUSSION THREAD

**Discussion by splendidcarter** [termux/termux-app#3061] [Q&A] [answered]
Hey how do you put in a command?

**Comment by sylirre** [comment, marked as answer]
Write a desired command using either a touch keyboard or a hardware one connected either via USB-OTG or Bluetooth to your device. Once the command was written, tap "Enter" and wait until it executes. Certain commands may ask additional input from user, so carefully read the output on terminal and proceed by instructions if any present.

Hopefully this explains how to put in a command...
